### PR TITLE
app not quitting when daemon start before the gui fix

### LINKED ISSUE #704: app not quitting when daemon started before
- start daemon
- launch app
- right click app in dock, Quit (mac) or close window
- app does not quit until we manually ctrl-C

### PR DESCRIPTION
**Pull request by vikas-cis**
#704

### CODE REVIEW

File: modules/daemon/daemon.js [hunk 1]
```diff
@@ -147,23 +147,18 @@ exports.check = function() {
 
 exports.stop = function() {
   return new Promise((resolve, reject) => {
-
-    if (daemon) {
```
Comment (pciavald) [inline modules/daemon/daemon.js]: indentation error after removing condition

Comment (pciavald) [inline modules/daemon/daemon.js]: indentation error after removing condition

File: modules/daemon/daemon.js [hunk 2]
```diff
@@ -147,23 +147,18 @@ exports.check = function() {
...
       rpc.call('stop', null, (error, response) => {
         if (error) {
           log.error('Calling SIGINT!');
           reject();
         } else {
           log.debug('Daemon stopping gracefully...');
```
Comment (pciavald) [inline modules/daemon/daemon.js]: daemon is not stopping gracefully if it was managed otherwise

**Comment by kewde** [comment]
I think this error is already fixed on `dev` by tecnovert?

Closing this PR for now, @pciavald please test this again to confirm, but it works just fine on my machine.

**Pull request closed without merging**